app/models/grda_warehouse/hud/project.rb

```diff
@@ -7,6 +7,8 @@ class Project < Base
     self.hud_key = :ProjectID
     acts_as_paranoid column: :DateDeleted
     has_paper_trail
+    
+    has_one :Project, through: :Affiliation
```

Comment:
@everestwitman I think this snuck in.  It shouldn't be here.  Can you remove it?